### DISCUSSION TITLE
Backup Home Assistant OS da Raspberry a Mini PC

### DISCUSSION THREAD

**Discussion by Stefano201** [indomus/forum#3130] [HUB: Home Assistant]
Buongiorno, ho intenzione di sostituire il mio vecchio rpi 4 con un mini pc (per la precisione dispongo di un BMAX con 8gb di ram Intel Celeron N4000 fino a 2,6 ghz).
Volevo sapere se una volta installato HAOS sul mini Pc posso tranquillamente caricare il backup creato dall'istanza installata sul Raspberry, senza problemi di compatibilità.

**Comment by snakuzzo** [comment]
Sì. Lo carichi e quando ha finito, ti consiglio di riavviare completamente l'host...perché spesso gli addon si incartano perché vengono tirati su come duplicati.

**Reply by Stefano201** [reply]
Grazie!
In realtà avrei voluto installare Core, ma utilizzo diversi add-on e non posso farne a meno.
